cli/flags/flags.go

```diff
@@ -136,14 +137,14 @@ func PrivateKeyFlag(c *cobra.Command) {
 	AddPersistentStringFlag(c, privKey, "", "Path to initiator Private Key file", false)
 }
 
-// GenerateInitiatorKeyFlag adds flag to generate a random secure password and initiator RSA key pair encrypted with this password
-func GenerateInitiatorKeyFlag(c *cobra.Command) {
-	AddPersistentBoolFlag(c, generateInitiatorKey, false, "Generates a random secure password and initiator RSA key pair encrypted with this password", false)
+// GenerateInitiatorKeyIfNotExistingFlag adds flag to generate a random secure password and initiator RSA key pair encrypted with this password
+func GenerateInitiatorKeyIfNotExistingFlag(c *cobra.Command) {
+	AddPersistentBoolFlag(c, generateInitiatorKeyIfNotExisting, false, "Generates a random secure password and initiator RSA key pair encrypted with this password", false)
 }
 
 // GetGenerateInitiatorKeyFlagValue gets flag to generate a random secure password and initiator RSA key pair encrypted with this password
 func GetGenerateInitiatorKeyFlagValue(c *cobra.Command) (bool, error) {
```

Comment:
just to say consistent this should be `GetGenerateInitiatorKeyIfNotExistingFlagValue` right?

Reply:
removed unused `Get`s